src/proof/lfsc_proof_printer.h

```diff
@@ -74,7 +74,35 @@ class LFSCProofPrinter
                                          std::ostream& out,
                                          std::ostream& paren);
 
+  /**
+   * The SAT solver is given a list of clauses.
+   * Assuming that each clause has alreay been individually proven,
+   * defines a proof of the input to the SAT solver.
+   *
+   * Returns the name of the LFSC value holding the proof, i.e. a value of type
+   * (clauses_hold ...)
+   *
+   * @param clauses The clauses to print a proof of
```

Comment:
Any assumption on the order of the clauses in this vector?

Reply:
Nope!

The function will work with any order. And in fact, the order won't influence the correctness of the proof, since this is for DRAT, where the working formula is a multiset of clauses (which has no concept of order!)

Reply:
Right, mixed it with lrat.